app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    get_command = command.split(" ")
+    if get_command[0] == "mv" and len(get_command) == 3:
+        command, old_file_name, new_file_name = get_command
```

Comment:
You already performed a command check, there is no need to allocate a variable for that value. Omit it using the following syntax:
```suggestion
        _, old_file_name, new_file_name = get_command
```